client-data/board.html

```diff
@@ -7,6 +7,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1" />
 	<link rel="stylesheet" type="text/css" href="board.css" />
 	<script src="/socket.io/socket.io.js"></script>
+	<script src="https://kit.fontawesome.com/0166a4ad2b.js" crossorigin="anonymous"></script>
```

Comment:
I would like to avoid requesting external assets...

Reply:
I see that you removed font-awesome in the past. I'm sure we can do without it.